lib/tasks/scheduler.rake

```diff
@@ -44,4 +44,11 @@ namespace :scheduler do
     ReschedulingJob.perform_later
     puts 'Rescheduling - done'
   end
+
+  desc 'Regenerate questionnaire headers'
+  task generate_questionnaire_headers: :environment do
```

Comment:
ding toegevoegd op Heroku en in de readme?

Reply:
Done en done.